chips/internal/_trailing-icon.scss

```diff
@@ -35,7 +35,7 @@
     width: var(--_icon-size);
   }
 
-  [name="trailing-icon"]::slotted(md-icon) {
+  [name="remove-trailing-icon"]::slotted(md-icon) {
      --md-icon-size: var(--_icon-size);
      color: inherit;
   }
```

Comment:
We should be able to remove all of this CSS by creating a wrapper `<span class="trailing icon">` around the slot. Then it will re-use shared `.icon` styles to size the icon:

```scss
.icon ::slotted(:first-child) {
  font-size: var(--_icon-size);
  height: var(--_icon-size);
  width: var(--_icon-size);
}
```

`color` inherits through slots, and `<md-icon>` explicitly has `color: inherit` so I don't think we need that declaration either.

Reply:
Nice, I just realize `font-size` on `md-icon` will size the icon to match this value, this is so perfect.